Add Payment (Current and Transactional) accounts (#374)
clearbank/clearbank.github.io: content/uk/docs/embedded-banking/payment-accounts.mdx

---
title: "Payment accounts"
metaTitle: "Payment accounts"
order: 6
showPageMenu: true
---

import EndpointBlock from "src/components/endpoint-block";
import Callout from "src/components/callout";

import WebhookPlaceholder from 'src/components/webhook-placeholder';
import WebhookSummary from 'src/components/webhook-summary';

## Payment accounts overview

<Callout colour="green">
<h3>Payment accounts are coming soon</h3>
<p>All information on this page describes a solution that is <strong>not yet live in production and therefore could change</strong>. This includes any endpoint and webhook details. The information is being provided early to aid you in integrating with our API in our UK simulation environment.</p>
</Callout>

> Payment accounts are regular GBP accounts with more payment functionality than savings accounts. With a payment account, your customers may set up direct debits, standing orders, and make day-to-day payments - among other features.

Reviewer: can they actually SET UP direct debits themselves? (sorry, I know this is fussy.) you or I, as normal consumers, can't set up a direct debit; we can however AGREE to have direct debits come out of our accounts.

Reviewer: Also, what're the max number of customers that can share an account? You don't specify

Author: might you say "participate in direct debits"? (?)

Agree on number of shared. I would imagine no different to savings

Reviewer: "With a payment account, your customers can have direct debits and standing orders, and make day‑to‑day payments — among other features." ? feel free to take or leave

Payment accounts come in two subtypes: **Current accounts** and **Transaction accounts**:

> * [Current accounts](#current-accounts) are fully featured but must comply with Payment Account Regulations (PARs) and Payment Service Regulations (PSRs).

Reviewer: ```suggestion
* [Current accounts](#current-accounts) have all available features, but must comply with Payment Account Regulations (PARs) and Payment Service Regulations (PSRs).
```

* [Transaction accounts](#transaction-accounts) have fewer features but are subject to lighter regulations (PSRs, but not PARs).

Payment accounts are available to embedded banking partners by prior agreement with ClearBank. If you're interested in using this account type, please contact your Client Director.

<Callout colour="blue">
  <strong>Note:</strong> Deposits held in Payment accounts may be eligible for FSCS protection. FSCS protection insures deposits of up to £120,000 per person per authorised firm (ClearBank) in the event of insolvency. Customers should review the specific terms and conditions of their account to confirm whether FSCS protection applies.
</Callout>

## Current accounts

> Current accounts provide extensive payment functionality for customers. Out of the box, they support:

Reviewer: ```suggestion
Current accounts provide extensive payment functionality for customers. As default, they support:
```

* Receipt of payments from other account holders and third parties
* Regular outbound payments via standing orders and direct debits

> * Anytime bank transfers

Reviewer: what is this? what is an "anytime bank transfer"? sending money via FPS?

Author: Legalese from product, I'll see if we can specify

* Open Banking
* Joint account ownership

> * Account switching

Reviewer: Is this CASS (current account switching service)? or is it another type of account switching? it might be worth getting clarification

Author: Agree, I asked the same. Clarification yielded that we should not (yet) specify
